docs/install/usb-install.md

```diff
@@ -30,6 +30,14 @@ Make sure you choose the correct device. The process erases data on the selected
 
 ## Common issues
 
+###
+
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know how to set the root partition and where to locate `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
```

Comment:
```suggestion
You can see the current root partition in the `grub.cfg`, once you've set the root to be within where the ISO lives on the USB. To view the `grub.cfg` from the ISO, run the command `cat /boot/grub2/grub.cfg`.
```

So to view it, the user will need to look at the `grub.cfg` with something like `cat` - sadly at least on our Grub 2.06, it seems `less` isn't an available command :sweat_smile:  - but! - `cat` should work great for viewing.